catalog/transformer/catalog-transformer-streaming-impl/src/main/java/org/codice/ddf/transformer/xml/streaming/impl/GmlHandler.java

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.transformer.xml.streaming.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.codice.ddf.transformer.xml.streaming.SaxEventHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xml.sax.Attributes;
+import org.xml.sax.Locator;
+import org.xml.sax.SAXException;
+
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.io.WKTWriter;
+import com.vividsolutions.jts.io.gml2.GMLHandler;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.AttributeImpl;
+
+/**
+ * A sax event handler to handle GML sax events and stores the data as a WKT. Delegates GML events to {@link com.vividsolutions.jts.io.gml2.GMLHandler}.
+ * Uses {@link com.vividsolutions.jts.io.WKTWriter} to write the Geometries to WKT.
+ * Note: ONLY CAN PARSE GML2 points. Will throw hard-to-debug Null Pointer Exceptions if used with GML3 or other GML2 geometries.
+ */
+public class GmlHandler implements SaxEventHandler {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GmlHandler.class);
+
+    private List<Attribute> attributes;
+
+    private boolean readingGml = false;
+
+    private static List<String> watchedElements = new ArrayList<>();
+
+    GMLHandler gh;
```

Comment:
These two classes have the same letters. Only the case of the "M" and the "L" are a different case. It looks like `GMLHandler` comes from a library, so that cannot be easily changed. Would it make sense to rename `GmlHandler` in some way that differentiates from the library GML handler class?